rust/lance/src/dataset/sql.rs

```diff
@@ -131,13 +144,52 @@ impl SqlQuery {
     }
 }
 
+fn contains_tokens() -> ScalarUDF {
```

Comment:
probably add some documentation explaining its usage and arguments?

Reply:
ok, let me fix it.